### ISSUE TITLE
Add OCaml examples

### ISSUE THREAD

**Issue by Drup** [gerdstolpmann/omake#24]
The manual is very precise and gives approximately zero intuition on how to build your own OMakefile from script. The only example is in 3.3 and is 1) using the many-OMakefile approach 2) not showing much.

If you are looking for examples to convert: https://github.com/ocaml/oasis/tree/master/examples

(And yes, I'm aware of the omake oasis backend, but It doesn't help to understand omake itself ..)

**Comment by neolem** [comment]
Indeed, I having similar experience while trying to use omake to do what I am achieving with Make and ocamlbuild. Maybe, some more examples and documentation targeted/geared towards developers who are familiar with ocamlbuild would be of immense help.
